direct/src/showbase/TkGlobal.py

```diff
@@ -31,7 +31,7 @@ def bordercolors(root, colorName):
         '#%04x%04x%04x' % (darkRGB[0], darkRGB[1], darkRGB[2])
     )
 
-Pmw.Color.bordercolors = bordercolors
+Pmw.Color.bordercolors = bordercolors  # type: ignore[attr-defined]
```

Comment:
What's with all the ignores for Pmw - can't we get mypy to recognise Pmw members somehow?  Or just blanket ignore all of Pmw in a config file?

Reply:
I did that, but for some reason the errors still pop up on macOS. I have no idea what's going on with that; it's probably worth digging into.